src/reducers/progress.js

```diff
@@ -1,4 +1,21 @@
 export default function progress(state = [], action) {
-  console.log(state, action);
+  switch(action.type) {
+    case 'ANSWER_QUESTION':
+      const questionId = action.questionId
+      const answerId = action.answerId
+
+      return Object.assign({}, state, {
```

Comment:
The state is an hash from the key `progress` that contains the `currentQuestion` and an array of `answers`. I think it's correct